demo/full/scripts/controllers/ProgressBar.jsx

```diff
@@ -97,8 +97,12 @@ class Progressbar extends React.Component {
       isLive,
       bufferGap,
       player,
+      hasManuallySeeked,
     } = this.props;
-    const seek = position => player.dispatch("SEEK", position);
+    const seek = position => {
+      hasManuallySeeked();
```

Comment:
something beginning by `has` would let us think that it has no side effect and returns a boolean
Here it is an action that does the opposite.

Reply:
Why not just providing `onSeek`?